examples/deploy/terraform/cluster/variables.tf

```diff
@@ -118,13 +118,18 @@ variable "use_fips_endpoint" {
   default     = false
 }
 
-variable "irsa_external_deployments_operator" {
+variable "external_deployments_operator" {
```

Comment:
fwiw, keeping this variable structure as input to `modules/external-deployments` might also be preferable to flattening it as you pass it through